bundle/cmake/DawnCloneRepository.cmake

```diff
@@ -0,0 +1,93 @@
+##===------------------------------------------------------------------------------*- CMake -*-===##
+##                         _       _
+##                        | |     | |
+##                    __ _| |_ ___| | __ _ _ __   __ _ 
+##                   / _` | __/ __| |/ _` | '_ \ / _` |
+##                  | (_| | || (__| | (_| | | | | (_| |
+##                   \__, |\__\___|_|\__,_|_| |_|\__, | - GridTools Clang DSL
+##                    __/ |                       __/ |
+##                   |___/                       |___/
+##
+##
+##  This file is distributed under the MIT License (MIT). 
+##  See LICENSE.txt for details.
+##
+##===------------------------------------------------------------------------------------------===##
+
+include(CMakeParseArguments)
+
+# dawn_clone_repository

```

Comment:
Again (move to MSBuild)

Reply:
this one is actually a copy of the one in msbuild. The problem is that if msbuild is not found, the bundle will first clone it, and then it can use all the msbuild infrastructure. But how do we clone msbuild if we dont have msbuild?